accounts/tests/test_spam.py

```diff
@@ -81,7 +81,8 @@ def __test_report_object(self, flag_type, object):
 
         # Now check that after the new flags an email was sent
         self.assertEqual(len(mail.outbox), 1)  # Notification email sent
-        self.assertTrue("[freesound] Spam/offensive report for user" in mail.outbox[0].subject)
+        self.assertTrue(settings.EMAIL_SUBJECT_USER_SPAM_REPORT in mail.outbox[0].subject and
```

Comment:
because these are tests, perhaps it makes better sense to list them as 2 separate asserts, so that if one fails we can easily see which one it is

Reply:
good point! Addressed in 9439414688155b780b9743e02530e6f0df673a9c